app/src/ui/dialog/dialog-options.html

```diff
@@ -8,7 +8,7 @@
     <template>
         <style include="shared"></style>
 
-        <pl-dialog open="{{ open }}" prevent-dismiss>
+        <pl-dialog open="{{ open }}" prevent-dismiss$="[[ preventDismiss ]]">
```

Comment:
This should be `prevent-dismiss="[[ preventDismiss ]]"`. `$=` is only needed for binding to attributes but since we're binding to the `preventDismiss` property of `pl-dialog` directly we should use a regular data binding. See [attribute binding](https://www.polymer-project.org/2.0/docs/devguide/data-binding#attribute-binding) vs [property binding](https://www.polymer-project.org/2.0/docs/devguide/data-binding#property-binding).